src/Modal/Modal.js

```diff
@@ -91,7 +91,12 @@ export function Modal({ children, title, animationDuration, showClose, onClose,
   };
 
   const scrollToTop = () => {
-    modalRef.current.scroll(0, 0);
+    // short delay to make sure backdrop is rendered
+    setTimeout(() => {
+      if (modalRef.current) {
+        modalRef.current.scroll(0, 0);
+      }
+    }, 1);
```

Comment:
This seems like an assumption that all rendering needs just a millisecond, which may not be true if a browser/phone/pc is very busy.

Reply:
That's true. Do we want to just keep the tick out? I think the safeguard against the ref possibly not being set is okay. Though, it's more of a scapegoat than anything.